package.json

```diff
@@ -12,39 +12,42 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "alt": "0.16.10",
-    "highlight.js": "8.8.0",
-    "hjs-webpack": "2.8.1",
-    "lodash": "3.9.3",
+    "alt": "0.17.8",
+    "highlight.js": "8.9.1",
+    "hjs-webpack": "3.0.0",
+    "lodash": "3.10.1",
     "normalize.css": "3.0.3",
-    "object-assign": "3.0.0",
-    "radium": "0.13.2",
-    "react": "0.13.3",
-    "react-router": "1.0.0-beta2",
-    "react-tween-state": "0.0.5",
-    "style-loader": "0.12.3",
-    "stylus-loader": "1.2.1",
-    "tween-functions": "1.0.2"
+    "radium": "0.14.2",
```

Comment:
@knowbody Can you please confirm that packman progress has animation of moving jaws when a slide changes wIth any of the following versions of radium: 0.13.2, 0.15.0, 0.15.1. I didn't narrow it down any more precise than that.